django_superform/forms.py

```diff
@@ -176,6 +176,12 @@ def __init__(self, *args, **kwargs):
         super(SuperFormMixin, self).__init__(*args, **kwargs)
         self._init_composite_fields()
 
+    def __iter__(self):
+        for name in self.fields:
+            yield self[name]
+        for name in self.composite_fields:
+            yield self[name]
+
```

Comment:
It looks like I'm loose right fields order here. If composite and common fields are interleaved each other in the form definition, the iterator will return them in the wrong order.